### DISCUSSION TITLE
background image not importing correctly

### DISCUSSION THREAD

**Discussion by kamesh-git** [vercel/next.js#38010] [Help]
This is one of the line in my index.js code
` <div className={style.bg_img}></div>`
**corresponding css code**
```
.bg_img{ 
    background: url('../public/flower/flowers.png') white fixed ;
    position: fixed;
    filter: blur(8px);
    top: 0;
    height: 100%;
    width: 100%;
    z-index: -1;

}

```
in the above line i'm setting a backround image for that div element and i'm importing the styles very correctly

when i host it in my localhost everything works good and when i inspect the element i'm getting this results
![image](https://user-images.githubusercontent.com/85233395/175760955-3a27706a-8b55-4202-b4bb-9435050a9047.png)

but when i host my page in vercel for production the backround image is not importing properly and i'm rendering with some different url
![image](https://user-images.githubusercontent.com/85233395/175760997-66ac5198-3aa0-48c7-9148-3c40b6b26212.png)

and other than index.js page the background works very well

**Comment by thrila** [comment]
did you get any solution? having the same problem but i hosted my project on github

**Comment by valcosmos** [comment]
I tried this, it does work. Hope it can help u.

![image](https://user-images.githubusercontent.com/53572196/180114573-c184ca16-d4d6-44e8-b346-b60fcf4b2f7b.png)

![image](https://user-images.githubusercontent.com/53572196/180114513-5e4f0de5-9937-404a-9cfb-16514c536437.png)

**Comment by thrila** [comment]
i tried that too it worked when my website wasnt live when i hosted it the background image wouldnt show up do you know ow to solve that? also all the other images ta=hat arent background images work fine

**Reply by valcosmos** [reply]
any error in console?

**Reply by thrila** [reply]
yeah `Failed to load resource: net::ERR_UNKNOWN_URL_SCHEME`  
and `GET webpack:///mini-css-extract-plugin/next/_next/static/media/eventsview.3f095fc3.png `

**Reply by thrila** [reply]
Fixed the image path and it worked

**Reply by aphrysof** [reply]
Having the same Issue
How did you change the image path

**Reply by vakulio** [reply]
Same problem. Please tell me how to solve it

**Reply by thrila** [reply]
is Your project Live as well?

**Reply by vakulio** [reply]
No. Migrated to Astro.js)

**Reply by thrila** [reply]
cool i'm happy for you anyways  about how i fixed this issue ... first of the problem .. i uploaded a website made with next js to github (hosted it on github)  the images didnt show up when i modified the image path to the repo file of the uploaded repo it worked!!!

**Comment by imthevk** [comment]
I faced the same problem when deploying website to vercel ,
By using this runtime config it sorted out.
import getConfig from 'next/config';
const { publicRuntimeConfig } = getConfig();

**Reply by rambharat-dwt** [reply]
where i import this

**Comment by SamKomesarook** [comment]
An alternative is to use inline styles in addition to the class:

```typescript
import img from '../assets/images/image.jpg'

//...

<div style={{
      backgroundImage: `url(${img.src})`,
    }}
className={style.bg_img}></div>

```

**Reply by 1Dedios** [reply]
I just used the style attribute as is with the import statement and it worked (I didn't need extra CSS styling). Thank you for this 🚀

**Comment by Prajwal-Pardhi** [comment]
Try changing the URL syntax, in my case '/' was missing at the beginning but still img is visible on localhost but not after hosting. Try changing url of the image by inspecting the page, maybe it could work.
